fedora_distro_aliases/__init__.py

```diff
@@ -100,7 +100,13 @@ def get_distro_aliases(cache: Optional[SaveLoad] = None):
     key = operator.attrgetter("major_version")
     for majorver, group in itertools.groupby(sorted(epel_branching, key=key), key=key):
         aliases[f"epel-{majorver}-all"] = minors = list(group)
-        aliases[f"epel-{majorver}-branched"] = [x for x in minors if "." in x.branch]
+        # For a certain period of time there can be two branched releases
+        # (e.g. epel-10.1 has just branched but epel-10.0 hasn't been archived yet),
+        # in such case pick the one with higher minor version
```

Comment:
I am not super familiar with the EPEL development, so I am curious what happens if, in this window that you are describing, I want to submit a new update for my EPEL package.

Do I only want to submit it into `epel-10.1` or also `epel-10.0`? If both, we should probably keep returning both?

Reply:
Hm, actually, in mock `epel-10` points to `10.2` repos and `rhel+epel-10` points to `10.0` repos. @praiskup is that expected? If so, will this last until `10.2` is branched? Should we then make the `epel-10-branched` alias resolve to `epel-10.0`?

Reply:
> For a certain period of time there can be two branched releases

For my normal "aliases" use case (getting updated leaf package to EPEL ASAP), it very much depends on how long _the period_ is.  If it is like `<=` 14 days, I don't care to only update epel-10.1 and epel-10.2 (and ignore epel-10.0), but if it is longer, I probably want to do three builds to deliver quickly.

Since there's no build inheritance, I doubt anyone would be updating `10.0` and `10.2` (ignoring `10.1`) or `10.0` and `10.1` (ignoring `10.2`).  Well, people could do these weird things, but that would be a very crazy and expensive back-patching work.  Such things are typically done with great care, and manually - not risking the aliases mess things up.

Reply:
Thank you very much for the discussion, and sorry for my absence. The consensus makes sense to me, so I am going to merge the PR as it is now.